orcid-web/src/main/resources/freemarker/includes/ng2_templates/member-details-ng2-template.ftl

```diff
@@ -45,61 +45,13 @@
 	                <div  *ngIf="currentMemberDetails.integrations">
 		                <div *ngFor="let integration of currentMemberDetails.integrations">
 		                    <p><b>{{integration.name}}</b>&nbsp;<em>{{integration.stage}}</em></p>
-		                    <div *ngIf="!newBadgesEnabled && integration.badgeAwarded">
```

Comment:
@DanielPalafox We dont want to remove the code right away, we want to hide it behind a togglz, so, please roll this code back

Reply:
Ok, I will roll back the code